frontend/utilities/endpoints.ts

```diff
@@ -82,7 +82,7 @@ export default {
   PERFORM_REQUIRED_PASSWORD_RESET: `/${API_VERSION}/fleet/perform_required_password_reset`,
   QUERIES: `/${API_VERSION}/fleet/queries`,
   RESET_PASSWORD: `/${API_VERSION}/fleet/reset_password`,
-  RUN_QUERY: `/${API_VERSION}/fleet/queries/run`,
+  LIVE_QUERY: `/${API_VERSION}/fleet/queries/run`,
```

Comment:
API uses `/run`, but we agreed on front end to use "live" for our naming convention since it matches the UI wording